app/src/components/custom/forms/RequestForm.tsx

```diff
@@ -134,6 +140,12 @@ const RequestForm = (props: RequestFormProps) => {
 
   return (
     <>
+      <DeletePrompt
+        deleteHandler={handleDelete}
+        title='Delete Request?'
+        blurb='Are you sure you want to delete this request?;;This action will hide the request from the requestor and may only be undone by an administrator.'
```

Comment:
You could also use a string array for blurb

Reply:
That would have been better. I will add this to the to-do list.